sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorLogExporter.cs

```diff
@@ -19,7 +19,7 @@ internal class AzureMonitorLogExporter : BaseExporter<LogRecord>
         private readonly AzureMonitorPersistentStorage? _persistentStorage;
         private AzureMonitorResource? _resource;
 
-        public AzureMonitorLogExporter(AzureMonitorExporterOptions options, TokenCredential? credential = null) : this(new AzureMonitorTransmitter(options, credential))
+        public AzureMonitorLogExporter(AzureMonitorExporterOptions options, TokenCredential? credential = null) : this(TransmitterFactory.Instance.Get(options, credential))
```

Comment:
what is the benefit of doing this?

Reply:
which _this_ are you referring to?


Today, if a user creates all three exporters, the SDK creates three Transmitters. 
Each transmitter would have it's own instance of RestClient, PersistantStorage, and Statsbeat.

We want to have only 1 instance of Transmitter per ConnectionString. 
For an ideal use case, 1 user = 1 connection string = 1 transmitter.

Reply:
Thanks.
we were duplicating statsbeat today, if i add logs, metrics and traces?

Reply:
Not yet because statsbeat isn't in use 😂
We're waiting on this change